README.md

```diff
@@ -20,3 +22,23 @@ Example using the [example_add_i32](https://github.com/stellar/rs-stellar-contra
 $ stellar-contract-cli invoke --file example_add_i32.wasm --fn add --arg i32:1 --arg i32:2
 i32:3
 ```
+
+### Calling a host function
+
+For both options below, the file should contain a JSON representation of the `SCVec` xdr object, which is passed directly to `invoke_function` in the host. The order of the items in the `SCVec` is specified below.
+
+#### Creating a contract
+
+```
+// args = SCVec(contractCode, salt, ed25519, signature)
+stellar-contract-cli invoke --file <JSONARGS> create-contract
```

Comment:
What does `--file <JSONARGS>` mean? It looks like it is JSON instead of a WASM file. We should avoid overloading a single argument as the behavior will be confusing to anyone who hasn't in detail read the docs for the command.

Reply:
All commands now take JSON strings as args.